nautobot/project-static/js/job_result.js

```diff
@@ -23,20 +23,22 @@ function updatePendingStatusLabel(status) {
     elem.text(status.label);
 }
 
-function updateLogTable(result_id) {
+function updateLogTable(result_id, qs) {
     // Calls `update_log_table` to refresh the jobs table from the `/log-table/` endpoint
     // Grab the query string from session storage and pass it through to the log table.
-    let qs = window.sessionStorage.getItem(session_key) || "";
+    if (!qs){
+        qs = window.sessionStorage.getItem(session_key) || "";
+    }
```

Comment:
This feels wrong given that the sessionStorage was introduced to save what page of results were being displayed (#1289). With this change, are we regressing on that issue? (I.e. if we're filtering logs *and* viewing page N of results, are both the filter and the page number being respected?

Reply:
On filtering it resets pagination, IMO it makes no sense to remain on the current pagination page when filtering cause we are not just filtering that page alone.
But after filtering is applied switching between filtered pages works well